src/util/builders/url/api.director.ts

```diff
@@ -49,10 +49,7 @@ export default class ApiURLDirector {
   }
 
   buildVisualEditorURL() {
-    return urlBuilder
-      .setDomain(this.baseDomain)
-      .setQueryParams({ action: 'visualeditor', mobileformat: 'html', format: 'json', paction: 'parse', formatversion: '2', page: '' })
-      .build(true)
+    return urlBuilder.setDomain(this.baseDomain).setQueryParams({ action: 'visualeditor', mobileformat: 'html', format: 'json', paction: 'parse', formatversion: '2' }).build(true)
   }
 
   buildArticleApiURL(articleId: string) {
```

Comment:
No reason to save `domain` in a variable here... variable name is anyway wrong. Actually by reading the code this function looks buggy.

Reply:
I optimized this by removing unnecessary `buildBaseArticleUR`